intro_to_r/intro_to_r.Rmd

```diff
@@ -66,7 +82,7 @@ boys to newborn girls, so he gathered the baptism records for children born in
 London for every year from 1629 to 1710.  We can take a look at the data by 
 typing its name into the console.
 
-```{r view-data, eval=FALSE}
+```{r view-data}
 arbuthnot
```

Comment:
If you're relying on dplyr, I'd consider working only with tbl_dfs, since they don't overwhelm you with printed output, and display the type of each column (which I think is super useful when you're learning). They also display the dimensions inline.